### DISCUSSION TITLE
Pimcore Academy: "Data Source" video not loading properly

### DISCUSSION THREAD

**Discussion by NiklasBr** [pimcore/pimcore#15016] [General]
Can't play the following video:
`/academy/course/Data-Importer~158351/chapter-158370`

All other videos in the chapter works.

![Screenshot 2023-04-24 at 13 06 54](https://user-images.githubusercontent.com/279826/233979094-7587dc82-dfcf-4d61-9ee3-8e772b472a89.png)

**Comment by fashxp** [comment]
For me the video plays without problems. Can you try delete your cookies?

**Reply by NiklasBr** [reply]
It only works in Chrome, not Firefox or Safari.

**Reply by fashxp** [reply]
for me, it also works in Firefox, don't have Safari.

**Reply by NiklasBr** [reply]
The browser console says:

> HTTP "Content-Type" of "video/x-matroska" is not supported. Load of media resource https://pimcore.com/academy/videos/50-data-import/video-thumb__13057__video/data-importer-2.mp4 failed.

Matroska video data can not be in a MP4 container. That could be it.

**Reply by fashxp** [reply]
hmm, strange. Why is it working for me? And most of the others as it seems...

**Reply by NiklasBr** [reply]
To me it looks like you named a MKV file MP4 which may or may not work since they are not compatible formats and containers. Maybe check the actual encoded file format (not just filename extension) with a tool like ffmpeg or exiftool and compare to the others which are working. Also, maybe you have some browser extension installed which makes it work?

**Reply by fashxp** [reply]
Hmm, the original file is a mkv indeed, but ffpmeg should do the transforming to mp4, right? 
I'm using a naked Firefox, so no extensions there. Are you using Mac? That might explain the difference of our systems. 

Now, I recreated the video thumbnail, maybe that helps?

**Reply by NiklasBr** [reply]
Still broken, and it is the only broken video on the Academy site I can find. The server keeps claiming that it is a Matroska video in an MP4 container (which we should expect not to work), see headers:

![Screenshot 2023-05-02 at 10 27 00](https://user-images.githubusercontent.com/279826/235617143-d21168f7-75cd-4397-a438-752de9ec64a4.png)

**Comment by alexz707** [comment]
> Still broken, and it is the only broken video on the Academy site I can find. The server keeps claiming that it is a Matroska video in an MP4 container (which we should expect not to work), see headers:
> 
> ![Screenshot 2023-05-02 at 10 27 00](https://user-images.githubusercontent.com/279826/235617143-d21168f7-75cd-4397-a438-752de9ec64a4.png)

@NiklasBr we have updated the video format and it should now work on Firefox on MacOS too. At least it is working on my Mac now.
Can you please test & confirm it?

Thanks a lot!
Best
Alex

**Reply by NiklasBr** [reply]
It's an actual MP4 content type now thanks!
